packages/components/src/Tree/TreeItem.tsx

```diff
@@ -195,7 +208,11 @@ const TreeItemLayout: FC<TreeItemProps> = ({
           {props.icon && (
             <PrimaryIcon name={props.icon} size={defaultIconSize} />
           )}
-          <FlexItem flex="1">{children}</FlexItem>
+          <FlexItem flex="1">
+            <Paragraph fontSize="xsmall">
+              <TextWrapper>{children as string}</TextWrapper>
```

Comment:
`Fragment` doesn't require `any` workaround.

```suggestion
              <TextWrapper>{children}</TextWrapper>
```